routes/api.js

```diff
@@ -0,0 +1,167 @@
+const express = require('express');
+const api = express.Router();
+const config = require('../config');
+const async = require('async');
+const stripe_secret = config.STRIPE_SECRET;
+const stripe_public = config.STRIPE_PUBLIC;
+const stripe = require('stripe')(stripe_secret);
+const uuid = require('uuid/v4');
+const bodyParser = require('body-parser');
+
+module.exports = function (db, redis) {
+  api.use(bodyParser.json());
+  api.use(bodyParser.urlencoded(
+  {
+    extended: true,
+  }));
+  api.route('/').get((req, res, next) => {
+    db.from('api_keys')
+      .where({
+        account_id: req.user.account_id
+      }).asCallback((err, results) => {
+          if (err) {
+            return next(err);
+          }
+          
+          if (results.length > 0) {
+            stripe.customers.retrieve(results[0].customer_id,
+            (err, customer) => {
+              if (err) {
+                return next(err);
+              }
+              
+              const source = customer.sources.data[0];
+              
+              return res.render('api', {
+                key: results.length > 0 ? results[0].api_key : null,
+                stripe_public,
+                credit_brand: source.brand,
+                credit_last4: source.last4
+              });
+            })
+          } else {
+            return res.render('api', {
```

Comment:
this isn't really an API if you're rendering HTML?

Reply:
It's rendering the Jade file.